src/lib/edge-functions/registry.mjs

```diff
@@ -548,7 +548,7 @@ export class EdgeFunctionsRegistry {
    * @returns {Promise<void>}
    */
   async #setupWatcherForDirectory(directory) {
-    const ignored = [new RegExp(this.#servePath)]
+    const ignored = [`${this.#servePath}/**`]
```

Comment:
is this using unix paths on windows as well, I guess so but just checking.

Reply:
Yes: https://github.com/micromatch/micromatch#backslashes.